### PR TITLE
Add Basic Scripting

### PR DESCRIPTION
**Pull request by JesterOrNot**

### CODE REVIEW

**Comment by JesterOrNot** [comment]
Fixes #203

File: main.go [hunk 1]
```diff
@@ -23,5 +24,11 @@ func main() {
 		} else {
 			shell.InvalidNumberOfArgs(os.Args[1])
 		}
+	} else if os.Args[1] == "run" {
+		f, _ := os.Open(os.Args[2])
+		scanner := bufio.NewScanner(f)
+		for scanner.Scan() {
+			shell.Evaluate(scanner.Text())
```
Comment (RDIL) [inline main.go]: Multiline script actions may break this

Comment (JesterOrNot) [reply main.go]: We don't have support for multi-line yet. Should we put this on hold till then?

Comment (JesterOrNot) [reply main.go]: Or maybe we could make it semicolon delimited

Comment (RDIL) [reply main.go]: Just write a parser that splits `\n`s into `; `, and have the runner split based on said dividers, and run instructions one at a time based on that

Comment (JesterOrNot) [reply main.go]: K

Comment (JesterOrNot) [reply main.go]: Fixed!

**Pull request merged**